File: c/DivideByThreeAndThirteen.c
#include<stdio.h>
int main(){
    int num;
    printf("Enter number: ");
    scanf("%d",&num);
    if(num % 3 == 0 && num % 13 == 0){
        printf("number is divisible by 3 and thirteen");
    }
    else{
        printf("number is not divisible by 3 and thirteen");
    }
    return 0;
}

File: kotlin/DivideByThreeAndThirteen.kt
import java.util.*
fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    print("Enter number: >")
    val num : Int= scanner.nextInt()
    
    if(num % 3 == 0 && num % 13 == 0){
        println("number is divisible")
    }
    else{
        println("number is not divisible")
    }
}